tests/contrib/django/test_django_appsec.py

```diff
@@ -380,3 +380,74 @@ def test_django_weak_hash(client, test_spans, tracer):
         vulnerability = json.loads(root_span.get_tag(IAST_JSON))["vulnerabilities"][0]
         assert vulnerability["location"]["path"].endswith("tests/contrib/django/views.py")
         assert vulnerability["evidence"]["value"] == "md5"
+
+
+def test_request_ipblock_nomatch_200(client, tracer):
+    with override_global_config(dict(_appsec_enabled=True)), override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)):
+        tracer._appsec_enabled = True
+        # Hack: need to pass an argument to configure so that the processors are recreated
+        tracer.configure(api_version="v0.4")
+
+        result = client.get("/?a=1&b&c=d", HTTP_X_REAL_IP="8.8.8.8")
+        assert result.status_code == 200
+        assert result.content == b"Hello, test app."
+
+
+def test_request_ipblock_match_403(client, test_spans, tracer):
```

Comment:
I wonder if we should make this a snapshot test as well. It would be good confirmation to know exactly which spans get created. Are only the code paths we want getting executed?

Reply:
(for all of these three new ones?)

Reply:
Good idea. Never did a snapshot test so I'll take a look at the existing ones.